libr/anal/p/anal_x86_cs.c

```diff
@@ -777,6 +777,31 @@ static void anop_esil(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len,
 		}
 		break;
 	case X86_INS_SHL:
+	{
```

Comment:
ths seems like spaguetti with the other PR, can you generalize the patch to reuse this switch case?